crypto/src/traits.rs

```diff
@@ -185,3 +191,20 @@ pub trait AggregateAuthenticator:
         messages: &[&[u8]],
     ) -> Result<(), Error>;
 }
+
+pub trait KeyBytes:
+    AsRef<[u8]>
+    + Display
+    + Serialize
+    + DeserializeOwned
+    + Copy
+    + Debug
+    + PartialEq
+    + Eq
+    + Hash
+    + PartialOrd
+    + Ord
+    + ToFromBytes
+    + FromStr
+{
+}
```

Comment:
Why? Can't you just derive most, if not all of these for `PublicKeyBytes`, which is essentially the only implementation we need?

Reply:
All of these are derived anyways, it's just that I needed the bounds for something earlier - anyways I'll remove the function that requires this for now.